pytorch_lightning/trainer/trainer.py

```diff
@@ -459,21 +457,17 @@ def fit(
         #                                |                                  ||
         #                         trainer.dispatch                          ||  LIGHTNING
         #                                |                                  ||
-        #    start_training or start_testing or start_predicting call       ||  FLOW
+        #    start_training or start_evaluating or start_predicting call    ||  FLOW
         #                        from `accelerator`                         ||
         #                                |                                  ||  DIRECTION
-        #             run_train or run_test or run_predict call             ||
+        #             run_train or run_evaluate or run_predict call         ||
         #                           from `trainer`                          ||
         #                                |                                  ||
         #                             results                               \/
         # This is used to guide readers to the core loops: train, test, predict.
         # `run_predict` is the simplest to understand, use `Go to Definition` to read it :)
-        # Search for `start_training` or `start_testing` or `start_predicting` in
+        # Search for `start_training` or `start_evaluating` or `start_predicting` in
         # `pytorch_lightning/plugins/training_type` folder to find accelerator dispatch functions.
-        self.accelerator.train_loop = self.run_train
-        self.accelerator.validation_loop = self.run_evaluation
-        self.accelerator.test_loop = self.run_evaluation
-        self.accelerator.predict_loop = self.run_predict
```

Comment:
What is it for? Maybe we should add a comment?

Reply:
I think @tchaton said for documentation purposes, so users who read the code here can click on the methods and inspect them. William said it must stay.

Reply:
That's funny because it really confused me, as it didn't make any sense after the accelerator refactor and I couldn't find any uses for it

Additionally, with the nice comment @tchaton added right above, I don't think it's needed

But i'll wait for @tchaton to make the call to keep/remove